jwala-services/src/test/java/com/cerner/jwala/service/jvm/impl/JvmServiceImplTest.java

```diff
@@ -1118,6 +1118,20 @@ public void testDeleteNonStoppedAndNotNewJvm() {
         verify(Config.mockJvmPersistenceService, never()).removeJvm(id);
     }
 
+
+    @Test
+    public void testDeleteFailedJvm() {
```

Comment:
Does the other delete method need a test? This method only tests the hardDelete

Reply:
The other deleteJvm() method is part of Version 2 service implementation and I don't see Junits /use cases on that .Hence, Skipped them for now.